### ISSUE TITLE
[4.0][Proposal] Remove BackpackUser in favour of User?

### ISSUE THREAD

**Issue by tabacitu** [Laravel-Backpack/Base#399]
Upon installing Backpack\Base, Backpack publishes our own model for the admin, ```BackpackUser```. This was done in order to:
1) make the installation as smooth as possible (before this we were asking the developer to add traits to their User model);
2) possibly allow the developer to add methods/attributes only to the admin;

It seems to me like (2) is rarely used, so it doesn't justify the added complexity for everybody. 
And (1) can be done in a different way:
- upon installation, look for ```App\User.php``` or ```App\Models\User.php``` - if it exists, use that; if not, ask the developer to punch in his custom user model;
- ask the developer if we can add a few traits to his User model (```CrudTrait``` and a trait that fixes the password reset for admins); if so - add those traits; if not, ask the developer to do it manually;

This breaking change should:
- make it _a little_ more difficult to install Backpack (reply y/n to prompt in most cases);
- make it easier to use packages like PermissionManager (see https://github.com/Laravel-Backpack/PermissionManager/issues/183);

Thoughts anyone?

**Comment by rodrigouriarte05** [comment]
Hi there, i think its confussing having two Users models. I agree it would be easier just leaving App/User.php

**Comment by danielwaghorn** [comment]
I agree on the removal, it's tripped me up before forgetting about it, and makes it a bit more complex when thinking about Guards.

**Comment by evanrthompson** [comment]
Initially I really liked the idea of separating my users from my admins but at the end of the day it just creates unnecessary complexity (particularly when dealing with permissions and guards). I'd be for relying on the User model by default 👍
